common/testing/testvars/any.go

```diff
@@ -80,6 +82,6 @@ func (a Any) ApplicationFailure() *failurepb.Failure {
 	}
 }
 
-func (a Any) RunID() string {
-	return uuid.New()
-}
+func (a Any) InfiniteTimeout() *durationpb.Duration {
```

Comment:
I have the same comment somewhere below.
Why adding this to Any? It is not random, it is very specific value.